### DISCUSSION TITLE
Timeline view should use local timezone, not browser timezone

### DISCUSSION THREAD

**Discussion by rknobbe** [blakeblackshear/frigate#14412] [General Support] [answered]
**Describe what you are trying to accomplish and why in non technical terms**
When I'm checking on my cameras from a remote location (like on travel for work), I'd like the timeline view in the review tab to reflect the timezone of the Frigate server, not the timezone of the browser.

**Describe the solution you'd like**

**Describe alternatives you've considered**
I've already got my cameras to render timestamps on the video streams, and my cameras are all synced to a local ntp server (as is my Frigate server).

**Additional context**

**Comment by NickM-27** [comment]
you need to set the `timezone` field in the config

**Reply by rknobbe** [reply]
great, thanks. I'll give that a try.

For others, it's in the ui: section of the config file.

**Reply by CzarDestructo** [reply]
> you need to set the `timezone` field in the config

Seems simple enough but I've tried it and it still doesn't work, keeps using browser local time:
```
ui:
  # Optional: Set a timezone to use in the UI (default: use browser local time)
  timezone: Europe/Athens
```

**Reply by rknobbe** [reply]
I'm not a yaml expert, but it works like this, perhaps needs the quotes

```
ui:
  timezone: "America/Los_Angeles"
```

**Reply by boonusss** [reply]
Tried with the quoted, still timeline shows browser time.
It's weird cause the logs in the UI appear to have the configured tz. Looks the issue is specific for the timeline

**Reply by CzarDestructo** [reply]
Same, appears to be a bug

**Reply by NickM-27** [reply, marked as answer]
this has been fixed for next 0.15 beta
